pkg/cert/legobridge/certificate_test.go

```diff
@@ -60,6 +63,33 @@ var _ = Describe("Certificate", func() {
 		Entry("ECDSA with wrong size", certcrypto.KeyType(""), api.ECDSAKeyAlgorithm, 511),
 	)
 
+	DescribeTable("NewCertificatePrivateKeyDefaults",
+		func(algorithmStr string, rsaKeySizeInt, ecdsaKeySizeInt int, expectedError error) {
```

Comment:
While only a callback function in a test, in general, the type of a variable should not be part of the variable name:
```suggestion
		func(algorithm string, rsaKeySize, ecdsaKeySize int, expectedError error) {
```